content/en/docs/deploying-a-container-image/_index.md

```diff
@@ -14,7 +14,15 @@ After we've familiarized ourselves with the platform, we are going to have a loo
 In OpenShift we have used the `<project>` identifier to select the correct project. Please use the same identifier in the context `<namespace>` to do the same for all upcoming labs. Ask your trainer if you want more information on that.
 {{% /onlyWhen %}}
 
-First, we are going to directly start a new Pod. For this we have to define our Kubernetes Pod resource definition. Create a new file `03_pod.yaml` with the following content:
+First, we are going to directly start a new Pod. For this we have to define our Kubernetes Pod resource definition. Create a new file `pod_awesome-app.yaml` with the following content:
+
+{{% onlyWhen openshift %}}
```

Comment:
For OpenShift the note is between '... following content **:**' and the effective content.
Add the hint after the content section?

Reply:
If the note is below, users will first try to create the file and probably see the note (too) late. I adjusted the phrase slightly to not confuse users where the content should be.